plugins/command-line/prism-command-line.js

```diff
@@ -79,6 +62,21 @@
 		}
 
 		var codeLines = env.code.split('\n');
+
+		/** @type {int[]} */
+		var continuationLineIdxs = commandLine.continuationLineIdxs = [];
+		var lineContinuationStr = pre.getAttribute('data-continuation-str');
+
+		// Identify code lines that are a continuation line and thus don't need
+		// a prompt
+		if (lineContinuationStr && codeLines.length > 1) {
+			for (var j = 1; j < codeLines.length; j++) {
+				if (codeLines.hasOwnProperty(j - 1) && codeLines[j - 1].endsWith(lineContinuationStr)) {
```

Comment:
[IE11 doesn't support `endsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith).

Also, we should probably support optional spaces after the line continuation string.

Reply:
Will fix the endsWith.

Not sure I agree about supporting optional spaces.  Pretty sure in bash if you have anything between the `\` and the `\n` it is not a continuation.  May well be the same for powershell but am not a powershell user.  The aim with one of these blocks is to be able to copy/paste the content into a terminal and run it so if it contains extra spaces that break the continuation they won't be able to paste it and run it.  If someone has some other shell with a continuation that needs a space at the end then `data-continuation-str` supports that.

Reply:
I mostly wanted to help users that type in text manually (not copy'n'pasting it), but you're right. Also, if user really need this feature, then people will request it and adding it later is pretty easy.